kafka/metadata.csv

```diff
@@ -38,7 +42,12 @@ kafka.producer.message_rate,gauge,10,message,second,Producer message rate.,1,kaf
 kafka.producer.io_wait,gauge,10,nanosecond,,Producer I/O wait time.,-1,kafka,prod io wait
 kafka.consumer.max_lag,gauge,10,offset,,Maximum consumer lag.,-1,kafka,con max lag
 kafka.consumer.fetch_rate,gauge,10,request,,The minimum rate at which the consumer sends fetch requests to a broker.,1,kafka,con min fetch
+kafka.consumer.fetch_size_avg,gauge,10,request,,The average number of bytes fetched per request for a specific topic.,1,kafka,con avg fetch size
+kafka.consumer.fetch_size_max,gauge,10,request,,The maximum number of bytes fetched per request for a specific topic.,1,kafka,con max fetch size
+kafka.consumer.bytes_consumed,gauge,10,byte,second,The average number of bytes consumed per second for a specific topic.,1,kafka,con bytes consumed
 kafka.consumer.bytes_in,gauge,10,byte,second,Consumer bytes in rate.,1,kafka,con bytes in
 kafka.consumer.messages_in,gauge,10,message,second,Rate of consumer message consumption.,1,kafka,con msg rate
 kafka.consumer.zookeeper_commits,gauge,10,write,second,Rate of offset commits to ZooKeeper.,1,kafka,zoo offset commit
 kafka.consumer.kafka_commits,gauge,10,write,second,Rate of offset commits to Kafka.,1,kafka,kafka offset commit
+kafka.consumer.records_consumed,gauge,10,records,sec,The average number of records consumed per second for a specific topic.,1,con rec consumed
+kafka.consumer.records_per_request_avg,10,records,request,The average number of records in each request for a specific topic.,1,rec per request
```

Comment:
a column is missing (the one containing `kafka` near the end) to both these lines